src/apps/reservation-list/utils/helpers.ts

```diff
@@ -13,24 +13,12 @@ export const sortByOldestPickupDeadline = (
   );
 };
 
-export const getReservedDigital = (list: ReservationType[]) => {
-  // Sorts by pickupDeadline, then title
-  return list
-    .filter(({ state }) => state === "reserved")
-    .sort(
-      (objA, objB) =>
-        new Date(objA.pickupDeadline || new Date()).getTime() -
-          new Date(objB.pickupDeadline || new Date()).getTime() ||
-        (objA.title || "").localeCompare(objB.title || "")
-    );
-};
-
-export const getReservedPhysical = (list: ReservationType[]) => {
-  return list
```

Comment:
These functions were not used and did too many things filtering and sorting which are non-obvious from the function name.